apps/server/src/infra/calendar/service/calendar.service.ts

```diff
@@ -43,6 +43,21 @@ export class CalendarService {
 			});
 	}
 
+	deleteEventsByScopeId(scopeId: EntityId): Promise<void> {
+		return this.delete(`/scopes/${scopeId}`, {
+			headers: {
+				Authorization: scopeId,
+				Accept: 'Application/json',
+			},
+			timeout: this.timeoutMs,
+		}).catch((error) => {
+			throw new InternalServerErrorException(
+				null,
+				ErrorUtils.createHttpExceptionOptions(error, 'CalendarService:deleteEvent')
+			);
+		});
```

Comment:
It seems that this code will fail only in case of a network communication error - please also verify if the returned HTTP status code is valid (I think the Calendar Service API should always return HTTP 204 No Content right?).

Reply:
Also please add a unit test for that case (when server is properly returning a response, but it's not valid from the client's point of view because of an invalid response code) - we have to make sure we cannot mark the data deletion as done until we receive the correct response from the Calendar Service.